Sources/Supporting Files/InputBarAccessoryView+Availability.swift

```diff
@@ -55,12 +55,13 @@ extension AutocompleteManager {
     public var isCaseSensitive: Bool {
         get { return false }
         set {
-            if isCaseSensitive {
+            if newValue {
```

Comment:
I think this change was merged to master on a different PR, can you bring this PR up to date?